index.html

```diff
@@ -26,49 +24,115 @@ <h1>
+        <p>
+          If you are interested in contributing a proposal to the WICG, please
+          see our
+          <a href="https://github.com/WICG/proposals">proposals repository</a>
+          on GitHub. The usual steps for incubating a proposal in the WICG is as
+          follows:
+        </p>
+        <ol>
+          <li>
+            <a href="https://github.com/WICG/proposals">Submit a proposal</a>
+            outlining your idea.
+          </li>
+          <li>Get feedback and improve your proposal.</li>
+          <li>Find collaborators and create a GitHub repository.</li>
+          <li>Work on your proposal and seek consensus from the community.</li>
           <li>
-            <a href="https://discourse.wicg.io/">Discourse</a>
+            Advocate for adoption of your proposal to the
+            <a href="https://www.w3.org/">W3C</a> or the
+            <a href="https://whatwg.org/">WHATWG</a> for standardization.
           </li>
-        </ul>
+        </ol>
       </section>
-      <section id="publications" class="collapsable accelerated">
-        <h2 class="opener">Getting started</h2>
-        <pre class="bash">
-npm install -g wicg
-wicg init "The Awesome API"</pre
-        >
+      <section id="incubations">
+        <h2>Active Incubations</h2>
+        <p>
+          There are <span class="incubation-count generated"></span> incubations
+          actively being worked on by the WICG community. The
+          <a href="https://en.wikipedia.org/wiki/Median">median</a> incubation
+          time is <span class="median-duration generated"></span>. The longest
+          running incubation is
+          <a class="longest-incubation-name generated"></a>, which has been
+          incubating for approximately
+          <span class="longest-incubation-duration generated"></span>. The
+          shortest incubation is
+          <a class="shortest-incubation-name generated"></a>, which has been
+          incubating for
+          <span class="shortest-incubation-duration generated"></span>.
+        </p>
+        <p>
+          Incubations are listed from most active participation to least active,
+          based on the community's interaction with issues and incoming commits.
+        </p>
+        <dl class="docslist multicol" id="incubation-list"></dl>
       </section>
-      <section id="publications" class="collapsable accelerated">
-        <h2 class="opener">
-          Incubations <span class="incubation-count"></span>
-        </h2>
-        <ul class="docslist multicol" id="incubation-list"></ul>
+      <section id="archived">
+        <h2>Archived incubations</h2>
+        <p>
+          The following
+          <span class="archived-count generated"></span> incubations have been
+          archived and are no longer actively being pursued by the WICG
+          community. This indicates that either they have successfully
+          progressed to the <a href="https://www.w3.org/">W3C</a> or the
```

Comment:
This should probably be clearer that they have progressed to "a REC track in a Working Group at the W3C" rather than just "W3C" - as is, it doesn't delineate enough between WICG (which is in W3C) and actual-standards-track-venue-in-W3C.

Reply:
Good point. Will fix.